src/reference/environments/example_envs.rst

```diff
@@ -0,0 +1,49 @@
+Example Environments
+====================
+
+A set of possible installations of Cylc:
+
+.. csv-table::
+   :header:  environment, cylc, cylc uiserver, rose
+
+   `Cylc 8 basic`_, ✔️, ,
+   `Cylc 8 with UI Server`_, ✔️,✔️,
+   `Cylc 8 with Rose`_, ✔️,✔️,✔️
+
+Example installation commands:
+
+.. code-block:: bash
+
+   conda env create -f path/to/env.yml --name "name of env"
+
+   # ..or if you haven't specified a python version in the enviroment file:
+   conda env create -f path/to/env.yml --name "name of env" python==<version>
```

Comment:
This conflicts with the advice in `cylc-8-basic-python-3.7`.

Can either specify the Python version in the environment OR on the CLI (unless the two versions exactly match).

Better for reproducibility to put it in the environment file.

```suggestion
```